third_party/terraform/resources/resource_google_project_iam_policy.go

```diff
@@ -88,8 +89,8 @@ func resourceGoogleProjectIamPolicyRead(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
-	// we only marshal the bindings, because only the bindings get set in the config
-	policyBytes, err := json.Marshal(&cloudresourcemanager.Policy{Bindings: policy.Bindings})
+	// we only marshal the bindings and audit configs, because only the bindings and audit configs get set in the config
```

Comment:
afaik we're covering 2/3 non-deprecated fields now (https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy) so this comment is a little outdated imo